src/views/viewer/ChannelView/ChannelAbout.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { useParams } from 'react-router'
+
+import { useChannel } from '@/api/hooks'
+import { languages } from '@/config/languages'
+import { Avatar, Button, Text } from '@/shared/components'
+
+import {
+  AvatarContainer,
+  Container,
+  Details,
+  DetailsContainer,
+  DetailsText,
+  Links,
+  LinksContainer,
+  StyledAvatar,
+  TextContainer,
+} from './ChannelAbout.style'
+
+export const ChannelAbout = () => {
+  const { id } = useParams()
+  const { channel } = useChannel(id)
+
+  return (
+    <Container>
+      <div>
+        <TextContainer>
+          <Text variant="h4">Description</Text>
+          <Text variant="body1" secondary>
+            {channel?.description}
+          </Text>
```

Comment:
Don't know what designers think about it but I think we should omit description if there is no any.
This looks little bit weird. 
![image](https://user-images.githubusercontent.com/51168865/125964544-b698307a-c68a-417d-92b7-e5c8978294b1.png)
I mean, you don't need to change that, just want to raise an issue